### PR TITLE
Release April 2022 k8s patches

### PR DESCRIPTION
**Pull request by kinarashah**

### CODE REVIEW

**Review by rancher-max — COMMENTED** [review]
We will have channels.yaml and channels-rke2.yaml updated today in dev-v2.6 with the non-rc versions of the rke2/k3s releases, so if we can hold on this until that happens that would be ideal

**Comment by kinarashah** [comment]
@rancher-max Yep! no rush, we're waiting for that - just getting PRs up for release so we can start reviewing and double check if anything else is missing.

**Review by rancher-max — DISMISSED** [review]
LGTM and has been validated for both rke2 and k3s

**Review by snasovich — DISMISSED** [review]

File: rke/k8s_version_info.go [hunk 1]
```diff
@@ -613,6 +629,14 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.3-patch0",
 			MinRKEVersion:     "1.3.3-rc0",
 		},
+		"v1.22.8-rancher1-1": {
+			MinRancherVersion: "2.6.3-patch0",
+			MinRKEVersion:     "1.3.3-rc0",
+		},
```
Comment (rayandas) [inline rke/k8s_version_info.go]: @kinarashah We decided to remove K8s march patches right? so 1.22.8-rancher1-1 shouldn’t be here.

**Review by rayandas — CHANGES_REQUESTED** [review]

File: rke/k8s_version_info.go (hunk 1 above)
Comment (kinarashah) [reply rke/k8s_version_info.go]: @rayandas good catch, thanks! fixing it

**Review by rayandas — APPROVED** [review]
LGTM

**Review by rancher-max — APPROVED** [review]
No changes made to k3s/rke2 in latest pushes -- still LGTM

**Pull request merged**